### PR TITLE
Adjust headers size & spacing

### PR DESCRIPTION
**Pull request by eduardopicolo-cb**
## What changed? Why?

Adjust all headings/subtitles and updates all pages to use `PageTitle` and `PageSubtitle` components

## How has it been tested?

- [x] Locally
- [x] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

### CODE REVIEW

File: src/components/ui/pageTitleText/index.tsx [hunk 1]
```diff
@@ -28,10 +31,34 @@ export interface PageTitleProps
...
+      return 'xs'
+    default:
+      return DEFAULT_PAGE_TITLE_SIZE
+  }
+}
+
 export const PageTitle = React.forwardRef<HTMLHeadingElement, PageTitleProps>(
   ({ className, children, as: Comp = 'h1', size, withoutBalancer, ...props }, ref) => {
+    const computedSize = size || getSizeFromComp(Comp)
+    console.log('computedSize:', computedSize)
```
Comment (twistershark) [inline src/components/ui/pageTitleText/index.tsx:55]: we don't need this

File: src/components/app/footer/index.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export async function Footer({ locale }: { locale: SupportedLocale }) {
       <div className="container">
         <div className="flex flex-col gap-9 lg:flex-row lg:justify-between">
           <div className="max-w-2xl space-y-8">
-            <PageTitle as="h6" className="text-left">
+            <PageTitle as="h6" className="text-left" size={DEFAULT_PAGE_TITLE_SIZE}>
```
Comment (twistershark) [inline src/components/app/footer/index.tsx:25]: As the `DEFAULT_PAGE_TITLE_SIZE` is the default prop value for size, do we still need to pass it here?

File: src/components/app/pageAbout/index.tsx [hunk 3]
```diff
@@ -22,7 +22,9 @@ export function PageAbout({ title, description }: { title: string; description:
           ].map(({ value, label }) => (
             <Card key={label}>
               <Card.Heading>
-                <PageTitle as="p">{value}</PageTitle>
+                <PageTitle as="p" size={DEFAULT_PAGE_TITLE_SIZE}>
```
Comment (twistershark) [inline src/components/app/pageAbout/index.tsx:25]: As the `DEFAULT_PAGE_TITLE_SIZE` is the default prop value for size, do we still need to pass it here?

File: src/components/app/footer/index.tsx (hunk 2 above)
Comment (eduardopicolo-cb) [reply src/components/app/footer/index.tsx:25]: Yes, needed to overwrite the `h6` size and have the same size as a `PageTitle` without being an `h1` tag

File: src/components/app/pageAbout/index.tsx (hunk 3 above)
Comment (eduardopicolo-cb) [reply src/components/app/pageAbout/index.tsx:25]: Yes, needed to overwrite the `p` size

**Review by twistershark — APPROVED** [review]

**Pull request merged**